ompi/mpi/fortran/use-mpi-f08/generate_bindings.py.in

```diff
@@ -0,0 +1,768 @@
+"""Fortran binding generation code.
```

Comment:
copyright.  also update description for case of a single big .in file for f08 prototypes